metabase/metabase
# Fix null ordinal value formatting

Closes https://github.com/metabase/metabase/issues/39558

### Description

Fixes null formatting on ordinal scales.

### How to verify

Create a question
```
select 1 x, 10 y
union all select null, 10
union all select 3, 10
```
Switch x-axis to Ordinal and ensure the null is formatted as `(empty)`

### Demo

<img width="938" alt="Screenshot 2024-03-08 at 7 20 27 PM" src="https://github.com/metabase/metabase/assets/14301985/736481d5-ee71-4f3e-8088-68c9e7bc0bb7">

### Checklist
